src/current/cockroachcloud/authorization.md

```diff
@@ -39,29 +44,33 @@ When a user is first added to an organization, they are granted the default role
 
 To learn more, refer to [Manage organization users](managing-access.html#manage-an-organizations-users).
 
-The following roles may be granted to {{ site.data.products.db }} organization users within a specific organization:
+The following {{ site.data.products.db }} organization roles can be granted:
 
 ### Organization Member
 
-This default role is granted to all organization users once they are invited. It grants no permissions to perform cluster or org actions.
+This default role is granted to all organization users once they are invited. It grants no permissions to perform cluster or organization actions.
 
 ### Org Administrator
```

Comment:
Is there a reason why we use "Organization Member" and "Org Administrator"? I feel like we should harmonize this to one version or the other.

Reply:
I think these are actually the role names in the UI. @abhinavg6 WDYT?